sdkutil/event.go

```diff
@@ -87,3 +104,172 @@ func GetString(attrs []sdk.Attribute, key string) (string, error) {
 	}
 	return "", ErrNotFound
 }
+
+// BlockID provides type to properly format print blockchain ID types.
+type BlockID struct {
+	Owner    *sdk.AccAddress
+	DSeq     *uint64
+	GSeq     *uint32
+	OSeq     *uint32
+	Provider *sdk.AccAddress
+}
```

Comment:
cool.  i think that this can maybe go in a package of its own, but for now put it in its own file - `object_id.go` or something.